pkg/apis/serving/v1alpha2/explainer_alibi.go

```diff
@@ -42,7 +43,7 @@ func (s *AlibiExplainerSpec) CreateExplainerContainer(modelName string, parallel
 		args = append(args, constants.ArgumentWorkers, strconv.Itoa(parallelism))
 	}
 	if s.StorageURI != "" {
-		args = append(args, "--storage_uri", constants.DefaultModelLocalMountPath)
+		args = append(args, "--storage_uri", fmt.Sprintf("%s/%s", constants.DefaultModelLocalMountPath, modelName))
```

Comment:
let's leave explainer as is because it is not supporting multi explainer model yet.